<?xml version="1.0"?>
<launch>

  <arg name="reference_frame" default="base_footprint"/>
  <arg name="reachability_map" default="tracbot_5"/>
  <arg name="reachability_message" default="tracbot_5_ros_message"/>
    

  <node name="transform_reachability_to_ros_message" pkg="reachability" type="transform_reachability_to_ros_message" respawn="false" output="screen">
    <param name="reference_frame" value="$(arg reference_frame)"/>
    <param name="reachability_map" value="$(arg reachability_map)"/>
    <param name="reachability_message" value="$(arg reachability_message)"/>	
  </node>


</launch>
